ThirdPartyNotices

```diff
@@ -9,7 +9,7 @@ may be either trademarks or registered trademarks of Microsoft in the United Sta
 The licenses for this project do not grant you rights to use any Microsoft names, logos, or trademarks.
 Microsoft's general trademark guidelines can be found at http://go.microsoft.com/fwlink/?LinkID=254653.
 
-Privacy information can be found at https://privacy.microsoft.com/en-us/
+Privacy information can be found at https://privacy.microsoft.com/
```

Comment:
Ok ... and there are also broken links in the repo that point to a `default.aspx` privacy page that lands on a 404. I'm going to close this PR and reopen with a set of changes that don't touch the privacy links. I'll set up an issue to cover the privacy link updates to get those broken ones (404) to the correct address.